src/components/Header/Header.jsx

```diff
@@ -12,7 +13,7 @@ export function Header() {
         <div className="hero-img-overlay flex-column">
             <h2 className="hero-heading">Your ultimate stop to buy all</h2>
             <h2 className="hero-heading"> badminton accessories</h2>
-            <button className="btn btn-outline-primary hero-cta text-bold-500 text-center" href="/">Shop Now</button>
+            <Link className="btn btn-outline-primary hero-cta text-bold-500 text-center" to="shopNow">Shop Now</Link>
```

Comment:
suggestion: you can avoid camelcase in `to` value of `<Link>` as its a url.

Reply:
Thanks for pointing it out will change it.